tests/data_layer/test_data_handle.py

```diff
@@ -579,7 +590,8 @@ def test_load_parameters_override(self, mock_store, mock_model):
 
         assert actual == expected
 
-    def test_load_parameters_override_ordered(self, mock_store, mock_model, parameter_spec):
+    @mark.skip(reason="TODO move narratives into sos_model")
```

Comment:
What is your thinking here @tomalrussell? Currently data_handle manages the overriding of the parameter values. Do you suggest we move this into a method on SosModel?

Reply:
Think this was just a misreading of how the fixtures are written - I'll unskip and test.